content-src/styles/_activity-stream.scss

```diff
@@ -162,6 +162,8 @@ input {
 @import '../components/DiscoveryStreamComponents/DSMessage/DSMessage';
 @import '../components/DiscoveryStreamImpressionStats/ImpressionStats';
 @import '../components/DiscoveryStreamComponents/DSEmptyState/DSEmptyState';
+@import '../components/DiscoveryStreamComponents/DSTextPromo/DSTextPromo';
+@import '../components/DiscoveryStreamComponents/DSChevronLink/DSChevronLink';
```

Comment:
DSChevronLink has scss only  and no corresponding jsx. We should either move these in _DSTextPromo.scss or move in common styles under content-src/styles

Reply:
In terms of organization I think it makes more sense to put all the components in a parallel structure even if they don't require JSX siblings. DSChevronLink is broken out because it can be used in other places than just DSTextPromo.

`content-src/styles` has no organizational concept of components that I can tell (we could start one, but I'm not sure what the benefit is of having component SCSS in 2 different places?).